dali/test/python/test_operator_jpeg_compression_distortion.py

```diff
@@ -24,10 +24,42 @@
 test_data_root = os.environ['DALI_EXTRA_PATH']
 images_dir = os.path.join(test_data_root, 'db', 'single', 'tiff')
 dump_images = False
+sequence_length = 10
 
-def _testimpl_jpeg_compression_distortion(batch_size, device, quality):
+
+def _testimpl_jpeg_compression_distortion(batch_size, device, quality, format):
+  # JpegCompressionDistortion for 'FHWC' GPU format is not yet implemented
+  if device == 'cpu' and format == 'FHWC': # TODO: remove device limitation
+    _testimpl_jpeg_compression_distortion_vid(batch_size, device, quality)
+  elif format != 'FHWC':
+    _testimpl_jpeg_compression_distortion_img(batch_size, device, quality)
```

Comment:
How about cutting some duplication here (or you can extract only the data loading part to a separate funcitons and reuse the rest of the pipeline and its exectution?
```suggestion
  @pipeline_def(batch_size=batch_size, num_threads=3, device_id=0)
  def jpeg_distortion_pipe_vid(device='cpu', quality=None, ):
  if device == 'cpu' and format == 'FHWC': # TODO: remove device limitation
    def get_data(sample_info):
      out = np.random.ranf(size=[sequence_length, 10, 11, 3]).astype(dtype = np.uint8)
      return out
    seq = fn.external_source(source=get_data, parallel=False, batch=False, layout='FHWC')
  elif format != 'FHWC':
    encoded, _ = fn.readers.file(file_root=images_dir)
    in_images = fn.decoders.image(encoded, device='cpu')
  if quality is None:
    quality = fn.random.uniform(range=[1, 99], dtype=types.INT32)
  out_tensors = fn.jpeg_compression_distortion(seq[0], quality=quality, device='cpu')
  return out_tensors

pipe = jpeg_distortion_pipe_vid(device=device, quality=quality,
                              batch_size=batch_size, num_threads=2, device_id=0)
pipe.build()

for _ in range(3):
  out = pipe.run()
```

Reply:
Will do, refactoring once I'll have it working